### ISSUE TITLE
Unable to Bulk Update a Matrix Person Attribute

### ISSUE THREAD

**Issue by leahjennings** [SparkDevNetwork/Rock#4419]
### Description
We have a matrix person attribute that we use to track the different people who baptized an individual. Since that number can be different each time, we use a matrix attribute to be able to dynamically track that per person.

We often have what we call a "Baptism Weekend", where dozens of people get baptized all at once, usually by the same person. When we try to bulk update the matrix attribute, nothing happens. Matrix attributes should be able to be bulk updated like all other attributes.

### Steps to Reproduce
1. Go to the demo site and login as the administrator
1. Go to General Settings > Attribute Matrix Templates
1. Create a new Matrix Template
  * Name: Baptizers
  * Item Attribute: 
    * Type: Person
    * Name: Baptizer
1. Go to General Settings > Person Attributes
1. Add an attribute of type _Matrix_, and select the _Baptizers_ matrix template in the dropdown, and change the Category to _Membership_
1. Go to the Group Viewer and select a group that has more than one person in it
1. Click the _Bulk Update_ truck icon
1. Expand the _Membership_ section
1. Click the radio button beside _Baptizers_
1. Click the _+_ button to add a new baptizer

![<a href="https://user-images.githubusercontent.com/66277785/93485054-a39bbc80-f8d0-11ea-8379-e983515ee047.gif">https://user-images.githubusercontent.com/66277785/93485054-a39bbc80-f8d0-11ea-8379-e983515ee047.gif</a> bulkupdateerror]()

**Expected behavior:**

I would expect that clicking the _+_ button in bulk update would allow me to add a new matrix item into that attribute, then complete the bulk update process.

**Actual behavior:**

Clicking the _+_ button does nothing. No errors in the console, no errors on screen, it just doesn't do anything.

### Versions
* **Rock Version:** Rock McKinley 11.0 (1.11.0.3)
* **Client Culture Setting:** en-US

**Comment by leahjennings** [comment]
Update on this, and not sure if this would make it a separate bug report or not. But I found that if you follow the steps above, even though the "Baptizers" attribute is empty and the confirmation message says it is _Clearing_ the attribute, it actually still creates an empty matrix and assigns that matrix id to each person being bulk updated instead of creating an individual matrix item for each person.

We found this because after we held a baptism weekend and had 18 people we needed to update. We bulk updated two other attributes and left the "Baptizers" attribute checked before proceeding. On the backend it created one matrix entity and tied that one matrix to each person being updated, but I would think it should have created 18 different matrix entities and assigned one to each person.
